### DISCUSSION TITLE
Icon data in devices.yaml

### DISCUSSION THREAD

**Discussion by jlazkano** [Koenkk/zigbee2mqtt#25108] [Devices]
Hello,

I notice that my devices.yaml file is growing in size:

```
$ du -h devices.yaml 
1,8M    devices.yaml
```

There is a image for every device:

```
icon: >-
    data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAYA
```

Is it possible to delete the icon info from this file?

Regards.

**Comment by Koenkk** [comment]
> Is it possible to delete the icon info from this file?

Yes, just delete it (it gets added when you press the localise device images button)
